core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/editor/document/DocumentStorageImpl.java

```diff
@@ -68,7 +68,7 @@ public void saveDocument(final EditorInput editorInput, @NotNull final Document
         file.updateContent(document.getContents()).then(new Operation<Void>() {
             @Override
             public void apply(Void arg) throws OperationException {
-                Log.debug(DocumentStorageImpl.class, "Document saved (" + file.getPath() + ").");
+                Log.debug(DocumentStorageImpl.class, "Document saved (" + file.getLocation() + ").");
```

Comment:
There is a code under hood:
```
    private static void log(Class<?> clazz, LogConfig.LogLevel logLevel, Object... args) {
        String prefix =
                new StringBuilder(logLevel.toString()).append(" (").append(clazz.getName()).append("): ").toString();

        for (Object o : args) {
            if (o instanceof String) {
                logToBrowser(logLevel, prefix + (String)o);
            } else if (o instanceof Throwable) {
                Throwable t = (Throwable)o;
                logToBrowser(logLevel, prefix + ExceptionUtils.getStackTraceAsString(t));
            } else if (o instanceof JavaScriptObject) {
                logToBrowser(logLevel, prefix + "(JSO below)");
                logToBrowser(logLevel, o);
            } else {
                logToBrowser(logLevel, prefix + (o != null ? o.toString() : "(null)"));
            }
        }
    }
```
for each object argument there will be another log calling and looks like this:

![browser log](https://api.monosnap.com/rpc/file/download?id=gGyxPFrG0nKdVSlHHEOmJxOfn013fl)